lib/material.dart

```diff
@@ -103,17 +103,28 @@ typedef void ListRenderer(dynamic obj, CoreElement element);
 typedef bool ListFilter(dynamic obj);
 typedef int ListSort<T>(T obj1, T obj2);
 
+abstract class MItem {
```

Comment:
Ah, ok, an MItem just provides some labelling info (not access to the DOM). Perhaps move out of the material library, so we don't need a ref to material from the model library?